### DISCUSSION TITLE
Text Size

### DISCUSSION THREAD

**Discussion by changho3995** [tughi/aggregator-android#67] [Ideas]
I want to change text size.

(I wish the image size was possible.)

ps. I've also posted it on Issue, but do you have any questions or suggestions? Or do you put it on Discussion?

**Comment by tughi** [comment]
What do you mean by `(I wish the image size was possible.)`?

**Reply by changho3995** [reply]
If the image is small, I wish there was a function to see it larger..^^

**Reply by changho3995** [reply]
![Screenshot_20230116_225843_Aggregator.jpg](https://user-images.githubusercontent.com/94268955/212695492-9af5eb05-4e0b-4b50-b13c-475da4d3e548.jpg)![Screenshot_20230116_225849_Whale.jpg](https://user-images.githubusercontent.com/94268955/212695507-5d12e1b9-03ab-43c2-9478-bb902225f161.jpg)

The image shown in the app is too small...

**Reply by tughi** [reply]
Hmmm... I am not sure if the feed uses the same image as the browser. Websites tend to provide smaller images in their feed.

What I am sure of is that Aggregator resizes an image only if it is bigger than the display.

**Reply by changho3995** [reply]
Checking it again, I think so.
By any chance... What if there was a function that allows you to see only the image in a larger size by touching the image?
I just want to see the image bigger.

**Reply by tughi** [reply]
It would be possible to open the image with the system browser...
